pandas/core/generic.py

```diff
@@ -3073,7 +3075,9 @@ def astype(self, dtype, copy=True, raise_on_error=True, **kwargs):
             the same type. Alternatively, use {col: dtype, ...}, where col is a
             column label and dtype is a numpy.dtype or Python type to cast one
             or more of the DataFrame's columns to column-specific types.
-        raise_on_error : raise on invalid input
+        errors : {'raise', 'ignore'}, default 'raise'
+            - ``raise`` : allow exceptions to be raised on invalid input
+            - ``ignore`` : suppress raising exceptions on invalid input
         kwargs : keyword arguments to pass on to the constructor
```

Comment:
I've just checked and the sublist is rendered fine with, or without a line between `ignore`
and kwargs. I can add an extra line if that is the convention,